functorch/csrc/OutOfPlacePlumbing.cpp

```diff
@@ -1643,11 +1643,11 @@ Tensor lowerToNextLayer<batch_rule_75_t,Tensor,const Tensor &, Dimname, c10::opt
   return makeBatched(std::get<0>(results), std::get<1>(results), cur_level);
 }
 
-typedef std::tuple<Tensor,c10::optional<int64_t>> (*batch_rule_76_t)(const Tensor &, c10::optional<int64_t>, const Tensor &, c10::optional<int64_t>, int64_t, const Tensor &, c10::optional<int64_t>);
+typedef std::tuple<Tensor,c10::optional<int64_t>> (*batch_rule_76_t)(const Tensor &, c10::optional<int64_t>, const Tensor &, c10::optional<int64_t>, int64_t, ScalarType);
```

Comment:
Would be nice if these rules also mentioned the list/type of operators they correspond to 🤔

Reply:
These are codegen'ed, actually. Not sure if you modified it manually :P 

I do agree that it would be nice if they mentioned the operator lol.

Reply:
Ouch. Not sure why it didn't work for me. I cleaned the previous build and built it again but it didn't work (undefined symbol for this instantiation). Had to do it manually and it worked.

Reply:
Well, ideally, we'd integrate it into our build system... But that's kinda hard/annoying, and has some other ramifications too...
 
so.... we just haven't done it :P